conformance: Github action error - bad Dockerfile name

```
Download action repository 'opencontainers/distribution-spec@master'
##[error]'Dockerfile.conformance' should be either '[path]/Dockerfile' or 'docker://image[:tag]'.
```

Will take a look at actions syntax and see what can be done

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,3 +1,5 @@
+# Note: this Dockerfile is used for running conformance tests
+
 # ---
 # Stage 1: Install certs and build conformance binary
 # ---

--- a/action.yml
+++ b/action.yml
@@ -8,4 +8,4 @@ inputs: {}
 outputs: {}
 runs:
   using: docker
-  image: Dockerfile.conformance
+  image: Dockerfile

--- a/conformance/README.md
+++ b/conformance/README.md
@@ -33,13 +33,13 @@ Note: for some registries, you may need to create `OCI_NAMESPACE` ahead of time.
 
 #### Container Image
 
-You may use [Dockerfile.conformance](./../Dockerfile.conformance) located at the
+You may use [Dockerfile](./../Dockerfile) located at the
 root of this repo to build a container image that contains the test binary.
 
 Example (using `docker`):
 ```
 # build the image
-(cd ../ && docker build -t conformance:latest -f Dockerfile.conformance .)
+(cd ../ && docker build -t conformance:latest -f Dockerfile .)
 
 # run the image
 docker run --rm \

